packages/wdio-utils/src/envDetector.js

```diff
@@ -38,31 +38,30 @@ function isW3C (capabilities) {
 }
 
 /**
- * check if session is run by Chromedriver
- * @param  {Object}  capabilities  caps of session response
- * @return {Boolean}               true if run by Chromedriver
+ * check if session is run by a chromium based driver (chromedriver, msedgedriver)
+ * @param  {Object}  capabilities  capabilities of session response
+ * @return {Boolean}               true if run by chromium based driver
  */
-function isChrome (caps) {
-    if (!caps) {
+function isChromium (capabilities) {
+    if (!capabilities) {
         return false
     }
     return (
-        Boolean(caps.chrome) ||
-        Boolean(caps['goog:chromeOptions'])
+        capabilities.browserName === ('chrome' || 'msedge' || 'MicrosoftEdge') ||
+        Boolean(capabilities['goog:chromeOptions'] || capabilities['ms:edgeOptions'])
     )
```

Comment:
webdriverEnvironmentDetector calls with capabilities requested by user which can be just the browserName.  Also user sets browserName to MicrosoftEdge but, session returns browserName msedge.  Weird but, handling both.   Also added ms:edgeOptions as requested to support devtools. 
One caveat I though of is that old versions of edge will pass the isChromium test.   I thought of maybe checking that the version is 79 but, user may not provide a version number.

Reply:
maybe we don't call isChromium in webDriverEnvironmentDetector and just do something like: `isChromium: capabilities.browserName('chrome' || 'MicrosoftEdge')`
then we could do a version number check in isChromium() since it would never have to process the user defined capabilities

Reply:
Let's keep `isChrome` for backwards compatibility. I guess the best would be an `isChromium` function that checks for `isChrome` and `ms:edge` capabilities.